package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "open-city-profile-ui",
-  "version": "1.24.0",
+  "version": "1.23.0",
```

Comment:
Looks like accidentally changed version here 🤔

Reply:
yes, probably merged a line in wrong way.